<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptSaver</class>
 <widget class="QDialog" name="ScriptSaver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="name_lineEdit">
     <property name="placeholderText">
      <string>File Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="author_lineEdit">
     <property name="placeholderText">
      <string>Author</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="version_spinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="version_label">
       <property name="text">
        <string>&amp;Version</string>
       </property>
       <property name="buddy">
        <cstring>version_spinBox</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="description_lineEdit">
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="type_lineEdit">
     <property name="placeholderText">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="menuEntry_radioButton">
       <property name="text">
        <string>Menu Entry</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
